### ISSUE TITLE
Not mobile friendly and more config options

### ISSUE THREAD

**Issue by mikecabana** [hggeorgiev/ngx-image-editor#12]
Spacing and styling are all off on mobile.

![ngx-image-editor-mobile](https://user-images.githubusercontent.com/1682519/39719865-86f0d33e-5208-11e8-88fb-70c68ca93bb9.PNG)

It would also be nice to have the ability to set static defaults so that the editor can only spit out specific images. For example, I want a user to upload a profile photo and edit it so it conforms to 1:1 aspect ratio. I realize you can do this already but you can't hide the aspect ratio chooser if there is only one to chose from. Also  it would be nice to control the background color as well as being to toggle certain functionalities such as the zoom slider or the inputs for canvas size and crop size.

**Comment by hggeorgiev** [comment]
Thanks for the suggestions, @mikecabana . Hardcoded color styles 
have been [removed](https://github.com/hggeorgiev/ngx-image-editor/blob/master/CHANGELOG.md) in v.1.4.1 onwards.  I have to document the classes so that custom styles can be easily applied. 
Once this is done, the the ability to toggle certain features will be underway.

**Comment by hggeorgiev** [comment]
If you don't have any other questions/comments, I'll go ahead and close the issue.

**Comment by mikecabana** [comment]
Sounds good!
